app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>CRUD44</title>
+  <title><%= @title || "Album Manager" %></title>
```

Comment:
here you can instead `yield(:title)` and in the other places have `content_for(:title) { 'Title!' }`